src/util/Recoil_batch.js

```diff
@@ -0,0 +1,16 @@
+// Default to a dummy "batch" implementation that just runs the callback
+function defaultNoopBatch(callback) {
+  callback();
```

Comment:
Is there a case where we would actually want to invoke the callback right away outside of a test context? Thinking this may hide bugs where the react-dom/react-native batching wasn't used by mistake....

We could throw an error here instead to surface the problem and make users provide their own `callback => callback()` function

Reply:
Good idea. Will add 👍